chat_shell/chat_shell/models/factory.py

```
@@ -91,6 +91,60 @@ def _mask_api_key(api_key: str) -> str:
+    httpx_client: httpx.AsyncClient | None = None
+    try:
+        if provider == "anthropic":
+            httpx_client = getattr(
+                getattr(model, "_async_client", None), "_client", None
+            )
+        elif provider == "openai":
+            httpx_client = getattr(
+                getattr(model, "root_async_client", None), "_client", None
+            )
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)